### PR TITLE
checkov: 2.0.1034 -> 2.0.1065

### PR DESCRIPTION
**Pull request by fabaff**
###### Description of changes
https://github.com/bridgecrewio/checkov/releases

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 168710` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>7 packages built:</summary>
  <ul>
    <li>checkov</li>
    <li>python310Packages.bc-python-hcl2</li>
    <li>python310Packages.cyclonedx-python-lib</li>
    <li>python310Packages.pycep-parser</li>
    <li>python39Packages.bc-python-hcl2</li>
    <li>python39Packages.cyclonedx-python-lib</li>
    <li>python39Packages.pycep-parser</li>
  </ul>
</details>

File: pkgs/development/python-modules/cyclonedx-python-lib/default.nix [hunk 1]
```diff
@@ -46,13 +47,21 @@ buildPythonPackage rec {
...
     "cyclonedx"
   ];
 
+ checkPhase = ''
+   runHook preCheck
+   # Tests require network access
+   rm tests/test_output_json.py
+   ${python.interpreter} -m unittest discover -s tests -v
+   runHook postCheck
+ '';
```
Comment (drewrisinger) [inline pkgs/development/python-modules/cyclonedx-python-lib/default.nix:63]: Why python unittest vs pytest? pytest should run unittest tests OK.

**Review by drewrisinger — CHANGES_REQUESTED** [review]
* [ ] Diff LGTM, **minus conversion from pytest -> python unittest**.
* [x] Commit LGTM
* [x] Builds via ``nix-review`` on x86-64-linux:

```
7 packages built:
checkov python310Packages.bc-python-hcl2 python310Packages.cyclonedx-python-lib python310Packages.pycep-parser python39Packages.bc-python-hcl2 python39Packages.cyclonedx-python-lib python39Packages.pycep-parser
```

File: pkgs/development/python-modules/cyclonedx-python-lib/default.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/development/python-modules/cyclonedx-python-lib/default.nix:63]: With 2.2.0 I'm no longer able to get the test to pass with `pytestCheckHook`. Not sure if it's a path issue or something else. I will re-check with the next release.

**Pull request merged**